### PR TITLE
Sasha/fix bugs

### PR DESCRIPTION
**Pull request by sasha-scale**
after my latest PR merge I noticed that pytest was broken for me for some reason, changing the import statement `from helpers import *` to a relative import fixed it for me, curious if anyone else has the same issue.

### CODE REVIEW

**Comment by jihan-yin** [comment]
Looks like the issue is from https://github.com/scaleapi/nucleus-python-client/commit/560d9c6c1f9f5c02554575c301e1c9aee231a7bc, @srikanth-scale added an `init.py` to `tests/` to make it a module, so we can no longer treat its files as individual packages. This is good practice so not really an issue, thanks for fixing the imports though!

File: nucleus/constants.py [hunk 1]
```diff
@@ -1,5 +1,4 @@
-# NUCLEUS_ENDPOINT = "https://api.scale.com/v1/nucleus"
-NUCLEUS_ENDPOINT = "http://localhost:3000/v1/nucleus"
+NUCLEUS_ENDPOINT = "https://api.scale.com/v1/nucleus"
```
Comment (jihan-yin) [inline nucleus/constants.py:1]: 🙈

File: nucleus/model.py [hunk 2]
```diff
@@ -1,6 +1,10 @@
 from typing import List, Optional, Dict, Union
 from .dataset import Dataset
-from .prediction import BoxPrediction, PolygonPrediction
+from .prediction import (
+    BoxPrediction,
+    PolygonPrediction,
+    SegmentationPrediction,
```
Comment (jihan-yin) [inline nucleus/model.py:6]: Curious how things were running before we added this import

**Review by jihan-yin — APPROVED** [review]
Lgtm, thanks for the fixes!

File: nucleus/constants.py (hunk 1 above)
Comment (sasha-scale) [reply nucleus/constants.py:1]: RIGHT

File: nucleus/model.py (hunk 2 above)
Comment (sasha-scale) [reply nucleus/model.py:6]: they weren't...

**Pull request merged**